ReadMe.rst

```diff
@@ -124,6 +131,29 @@ This is provided by the `.github/workflows/Stale.yml` reusable workflow.
 Individual repositories can control the label and time settings but it is strongly recommended to use the default
 values defined in the reusable workflow for consistency.
 
+Release Drafting
+----------------
+
+In order to ensure semantic versioned is followed based on well-defined labels used in Project Mu pull requests, the
```

Comment:
```suggestion
In order to ensure semantic versioning is followed based on well-defined labels used in Project Mu pull requests, the
```

Reply:
Update in PR commit https://github.com/microsoft/mu_devops/pull/49/commits/b445c98d20c87a5e36f27d8743aa98b44ca2795d

Reply:
Updated in PR commit https://github.com/microsoft/mu_devops/pull/49/commits/b445c98d20c87a5e36f27d8743aa98b44ca2795d